api/api/models/fields.py

```diff
@@ -0,0 +1,21 @@
+from django import forms
+from django.core import validators
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+
+class URLTextField(models.TextField):
+    """URL field which uses the underlying Postgres TEXT column type."""
+
+    default_validators = [validators.URLValidator()]
```

Comment:
_(Up front context) I know we currently use a URL field for some of these (though this would be a new detail for `creator_url`, I think), so this is only relevant feedback for this PR due to the need to add new code to continue supporting it. Just wanting to clarify up front I know this isn't a decision you're making about the model, and that my intention is to ask whether we need this at all, and if not, then to give feedback that this code is unnecessary and can be removed._

---

This validator is the only thing that differentiates the URL field from a regular text field. I question whether we need this. Not blocking because it's a trivial thing to remove in the future or as a fast-follow if we like (would be a no-SQL migration). I'd prefer we didn't add this code, however. These write-time Django validators are irrelevant for our domain and usage of the ORM because we never write to these tables with the Django ORM (except in tests). Data validation either has to happen in the catalogue, data refresh, or not at all, but definitely not in Django write-time validators.

The change requested would be to use a `TextField` and forego write-time validators as a code-quality improvement and clarification of the intention of these models and the domain they're actually concerned with (i.e. _reading_).

Reply:
That's an interesting point. I can't think of a use case right now, but I also think it doesn't hurt to leave the validation for this kind of field. I have no strong opinion on either side.

Reply:
@AetherUnbound Just checking, were you able to create the issue?

@krysal I guess generally there is a rationale of removing code that isn't used. All code is a liability, either as a vulnerability or increased maintenance cost (the latter being most relevant and in fact exemplified here), so as much of it as we don't need is a good idea to remove.

Reply:
Created in https://github.com/WordPress/openverse/issues/4320